masterlist.yaml

```
@@ -4236,6 +4236,9 @@ plugins:
       - <<: *patchIncluded
         subs: [ 'DLC Shivering Isles' ]
         condition: 'active("DLCShiveringIsles.esp") and not active("All Natural - SI.esp")'
+      - <<: *patchIncluded
+        subs: [ 'Kvatch Rebuilt' ]
+        condition: 'active("Kvatch Rebuilt.esp") and not active("Kvatch Rebuilt Weather Patch.esp"'
```

```suggestion
        condition: 'active("Kvatch Rebuilt.esp") and not active("Kvatch Rebuilt Weather Patch.esp")'
```

The condition is missing a closing parenthesis.

---

Oops, dumb mistake. 🤦‍♂️